src/domain/occurrence/eventOccurrenceForm/ValidationSchema.ts

```diff
@@ -1,8 +1,7 @@
-import isBefore from 'date-fns/isBefore';
-import isFuture from 'date-fns/isFuture';
-import parseDate from 'date-fns/parse';
+import { isBefore, isToday, parse as parseDate } from 'date-fns';
```

Comment:
Is this going to increase the bundle size? These functions were imported line by line for a reason but I'm not sure id date-fns is tree shakeable

Reply:
+ isToday seems to be unused

Reply:
maybe we should make sure that we have imported these individually everywhere and test the bundle sizde with:

https://create-react-app.dev/docs/analyzing-the-bundle-size/